hedyorg/hedy
# 🪲 Improve error message for missing list in add-to and remove-from commands

This PR adds a specific error message when the user forgot to add a list at the end of the add-to and remove-from commands. Previously, we displayed a generic error that something is off with the line:
_We detected that Add is being used on line 3 which is not allowed. Can you try looking for a missing or an extra character on your code?_

Now we changed it to: _We detected that part of the code seems to be missing from the Add on line 3. Can you try adding the what's missing_

Fixes #5871

**How to test**
Go to level 3 and execute the following commands:
```
houses is Gryffindor, Hufflepuf, Ravenclaw, Slytherin
dislike is Slytherin
remove dislike from
```
and
```
houses is Gryffindor, Hufflepuf, Ravenclaw, Slytherin
dislike is Slytherin
remove dislike
```
Check that you agree with the error messages.
